test/integration/about_test.rb

```diff
@@ -5,10 +5,10 @@ class AboutIntegrationTest < IntegrationTestWithJavascript
   #   AboutIntegrationTest.test_0002_about page proxies should have version
 
   setup do
-    ComputeResource.any_instance.expects(:ping).at_least_once.returns([])
+    ComputeResource.any_instance.stubs(:ping).at_least_once.returns([])
     proxy_status = mock('ProxyStatus::Version')
-    proxy_status.expects(:version).at_least_once.returns('version' => '1.13.0')
-    SmartProxy.any_instance.expects(:statuses).at_least_once.returns(:version => proxy_status)
+    proxy_status.stubs(:version).at_least_once.returns('version' => '1.13.0')
+    SmartProxy.any_instance.stubs(:statuses).at_least_once.returns(:version => proxy_status)
```

Comment:
Something I just noticed now. It might be better to also get rid of the `.at_least_once`. It doesn't seem to hurt, but if it's not necessary.